emily/lambda/src/errors.rs

```diff
@@ -0,0 +1,166 @@
+use std::fmt;
+use aws_lambda_events::apigw::ApiGatewayProxyResponse;
+use aws_lambda_events::encodings::Body;
+use emily::models;
+use crate::common;
+
+#[derive(Debug)]
+pub enum EmilyApiError {
+    BadRequest(String),
```

Comment:
You could also use `thiserror` here to generate an `std::error::Error` implementation. This would make this code look like this:
```rust
#[derive(Debug, thiserror::Error)]
pub enum EmilyApiError {
    #[error("Bad Request: {0}")]
    BadRequest(String),
```
And then we could get rid of the manual `std::error::Error` and `fmt::Display` implementations.